src/main/java/org/scijava/display/DefaultDisplayService.java

```diff
@@ -219,20 +221,28 @@ public boolean isUniqueName(final String name) {
 
 	@Override
 	public Display<?> createDisplayQuietly(final Object o) {
+		final List<Display<?>> matchingDisplays = getMatchingDisplays(o);
+		if(matchingDisplays.size() > 0) {
+			// use the display with the highest priority
+			Display<?> display = matchingDisplays.stream().max(Comparator.comparing(Prioritized::getPriority)).get();
```

Comment:
I don't think you need this. The displays should already always be returned from the `PluginService` in high-to-low priority. Did you observe otherwise?